pinot-spi/src/main/java/org/apache/pinot/spi/config/table/UpsertConfig.java

```diff
@@ -19,15 +19,17 @@
 package org.apache.pinot.spi.config.table;
 
 import com.fasterxml.jackson.annotation.JsonPropertyDescription;
-import java.util.Collections;
+import com.google.common.base.Preconditions;
 import java.util.List;
 import java.util.Map;
 import javax.annotation.Nullable;
 import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
 import org.apache.pinot.spi.config.BaseJsonConfig;
+import org.apache.pinot.spi.utils.Enablement;
 
 
-/** Class representing upsert configuration of a table. */
+/// Class representing upsert configuration of a table.
```

Comment:
nit: rm one `/`

Reply:
This is the new javadoc format (3 '/' instead of 2) which is more concise